src/auth_api/endpoints/oidc.py

```diff
@@ -206,6 +210,9 @@ def handle_request(
             company=company,
         )
 
+        logger.log(message=f"User {oidc_token.ssn}", actor=oidc_token.ssn, 
+                   subject=oidc_token.subject)
+
```

Comment:
NO NO NO NO NO!!!
Never log SSN!!!! -> CPR

Better place would be in the orchestrator 
src/auth_api/orchestrator.py line:286 here we have both the correct actor and subject ids.